### PR TITLE
For Ember Times No. 98: Add Ember Focus Trap + Contributors

### PR DESCRIPTION
**Pull request by jayjayjpg**

### CODE REVIEW

File: source/2019-05-12-the-ember-times-issue-98.md [hunk 1]
```diff
@@ -127,9 +131,20 @@ We encourage you to [read and share the blog post](https://medium.com/ember-ish/
 
 ---
 
+## [Intentional Element Focus with Ember Focus Trap 💥](https://twitter.com/josemarluedke/status/1127967059415953408)
+
+This week the **Global Accessibility Awareness Day (GAAD)** on May 16 inspired developers all around the world to reflect on the way they use and build applications for the web. In her [recent call-to-action for GAAD](https://blog.emberjs.com/2019/05/13/global-accessibility-awareness-day.html) 📣, [@melsumner](https://github.com/MelSumner) encouraged the Ember community to take the time to browse the web using only their keyboard and with the support of assistive technologies. Or to spend time to improve their own projects in regards to accessibility.
```
Comment (ijlee2) [inline source/2019-05-12-the-ember-times-issue-98.md]: May I suggest the following for clarity?
```suggestion
On May 16, the **Global Accessibility Awareness Day (GAAD)** inspired developers all around the world to reflect on the way they use and build applications for the web. In her [recent call-to-action for GAAD](https://blog.emberjs.com/2019/05/13/global-accessibility-awareness-day.html) 📣, [@melsumner](https://github.com/MelSumner) encouraged the Ember community to take the time to browse the web using keyboard or an assistive technology, and to spend time to improve accessibility in their own projects.
```

File: source/2019-05-12-the-ember-times-issue-98.md [hunk 2]
```diff
@@ -127,9 +131,20 @@ We encourage you to [read and share the blog post](https://medium.com/ember-ish/
...
+
+Addon author [@josemarluedke](https://github.com/josemarluedke) followed this call-to-action and just published [Ember Focus Trap](https://github.com/josemarluedke/ember-focus-trap)! It's based on the JavaScript library [Focus Trap](https://github.com/davidtheclark/focus-trap) that helps you to intentionally trap focus in certain DOM nodes. This makes it easy for you to improve the keyboard accessibility of otherwise inaccessible elements such as modal dialogs.
+
+And with **Ember Focus Trap** adding focus to interactive elements in your Ember app is only an `ember install` away. Check it [out today](https://josemarluedke.github.io/ember-focus-trap/)!
```
Comment (ijlee2) [inline source/2019-05-12-the-ember-times-issue-98.md]:
```suggestion
With **Ember Focus Trap**, adding focus to interactive elements in your Ember app is only an `ember install` away. Check it [out today](https://josemarluedke.github.io/ember-focus-trap/)!
```

**Review by ijlee2 — APPROVED** [review]
Thank you for covering this important news!

**Comment by jayjayjpg** [comment]
Thank you for the useful suggestions @ijlee2 🙇‍♀

**Pull request merged**